### ISSUE TITLE
ODFW references from @aliwhitman

### ISSUE THREAD

**Issue by kellijohnson-NOAA** [nwfsc-assess/sa4ss#35]
Hi @aliwhitman. The sa4ss repository holds a template for creating stock assessment documents for the NW and SW Fisheries Science Centers. We are working on improving our workflow and given that you are such an integrated part of so many assessments, I am wondering if you can provide a master list of ODFW references that you cite when providing information for the stock assessments? Preferably this would be in the format of a .bib file, but I am happy to take contributions in any form. The goal here is to have a single master list of citations such that individual authors do not need to worry about compiling references, just citing them. I am happy to provide more information if needed. Thank you for all of your help this assessment season!

**Comment by aliwhitman** [comment]
Yes, that's a great idea!  There are a number of references that are typically used in indices and data summaries that I could easily grab in a bib file for you.  Some might overlap with the existing list too.  We also have a publications page for MRP where ODFW reports are "published" (https://www.dfw.state.or.us/MRP/publications/).

- [ ]  @aliwhitman will develop a bib file with the typical ODFW references used for assessments (later today, after some vermilion work!)
